index.bs

```diff
@@ -9815,6 +9815,24 @@ The method is invoked with the following arguments:
 	<a>computedNumberOfChannels</a> only when the node has single
 	output.
 
+	<div algorithm="check-process-outputs">
+		Let <var>outputs_in</var> be the <code>outputs</code> passed
+		to <code>process()</code>.  let <var>outputs_out</var> be the
+		<code>outputs</code> parameter on return from
+		<code>process()</code>.
+
+		1. Check <var>outputs_out</var>:
```

Comment:
I find this a bit weird. Can you lay out the steps normally with `if` and `and` statements?

Reply:
Done, I think.  Is this better?